nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -38,133 +79,6 @@ In addition to numerous new and upgraded packages, this release has the followin
   for a transition period so that in time the ecosystem can switch without
   breaking compatibility with any supported NixOS release.
 
-- `emacs` enables native compilation which means:
-  - emacs packages from nixpkgs, builtin or not, will do native compilation ahead of time so you can enjoy the benefit of native compilation without compiling them on you machine;
-  - emacs packages from somewhere else, e.g. `package-install`, will do asynchronously deferred native compilation. If you do not want this, maybe to avoid CPU consumption for compilation, you can use `(setq native-comp-deferred-compilation nil)` to disable it while still enjoy the benefit of native compilation for packages from nixpkgs.
-
-- `nixos-generate-config` now generates configurations that can be built in pure
-  mode. This is achieved by setting the new `nixpkgs.hostPlatform` option.
-
-  You may have to unset the `system` parameter in `lib.nixosSystem`, or similarly
-  remove definitions of the `nixpkgs.{system,localSystem,crossSystem}` options.
-
-  Alternatively, you can remove the `hostPlatform` line and use NixOS like you
-  would in NixOS 22.05 and earlier.
-
-- PHP now defaults to PHP 8.1, updated from 8.0.
-
-- PHP is now built `NTS` (Non-Thread Safe) style by default, for Apache and
-  `mod_php` usage we still enable `ZTS` (Zend Thread Safe). This has been a
-  common practice for a long time in other distributions.
-
-- PHP 8.2.0 RC 7 is available.
-
-- `protonup` has been aliased to and replaced by `protonup-ng` due to upstream not maintaining it.
-
-- Perl has been updated to 5.36, and its core module `HTTP::Tiny` was patched to verify SSL/TLS certificates by default.
-
-- Improved performances of `lib.closePropagation` which was previously quadratic. This is used in e.g. `ghcWithPackages`. Please see backward incompatibilities notes below.
-
-- Cinnamon has been updated to 5.4. While at it, the cinnamon module now defaults to
-  blueman as bluetooth manager and slick-greeter as lightdm greeter to match upstream.
-
-- OpenSSL now defaults to OpenSSL 3, updated from 1.1.1.
-
-- An image configuration and generator has been added for Linode images, largely based on the present GCE configuration and image.
-
-- `hardware.nvidia` has a new option `open` that can be used to opt in the opensource version of NVIDIA kernel driver. Note that the driver's support for GeForce and Workstation GPUs is still alpha quality, see [NVIDIA Releases Open-Source GPU Kernel Modules](https://developer.nvidia.com/blog/nvidia-releases-open-source-gpu-kernel-modules/) for the official announcement.
-
-<!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
-
-## New Services {#sec-release-22.11-new-services}
```

Comment:
Would it make sense to just mention how many services has been added in the last release and than maybe mention a couple interesting ones?

Reply:
Moved to the end, not sure if we want to do this given time constraints. cc @mweinelt